ziti/edge/conn.go

```diff
@@ -153,14 +153,23 @@ func (ec *MsgChannel) WriteTraced(data []byte, msgUUID []byte, hdrs map[int32][]
 	copyBuf := make([]byte, len(data))
 	copy(copyBuf, data)
 
-	msg := NewDataMsg(ec.id, ec.msgIdSeq.Next(), copyBuf)
+	seq := ec.msgIdSeq.Next()
+	msg := NewDataMsg(ec.id, seq, copyBuf)
 	if msgUUID != nil {
 		msg.Headers[UUIDHeader] = msgUUID
 	}
 
 	for k, v := range hdrs {
 		msg.Headers[k] = v
 	}
```

Comment:
Could we send this back on the connect reply instead?

Reply:
connect reply goes to ER/controller and not go the peer SDK